src/debugger/debugAdapterFactory.ts

```diff
@@ -102,7 +102,7 @@ export class LLDBDebugConfigurationProvider implements vscode.DebugConfiguration
         }
 
         // Delegate to CodeLLDB if that's the debug adapter we have selected
-        if (DebugAdapter.getDebugAdapterType(this.swiftVersion) === "lldb-vscode") {
+        if (DebugAdapter.getDebugAdapterType(this.swiftVersion) === "lldb") {
```

Comment:
again constant would just mean less places to change in the future

Reply:
done